assets/pages/jobdetails.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js">
+	<!--<![endif]-->
+	<head>
+		<meta charset="utf-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<title>Job Details</title>
+		<meta name="description" content="" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+
+		<!-- Compressed CSS -->
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/foundation-sites@6.6.3/dist/css/foundation.min.css" integrity="sha256-ogmFxjqiTMnZhxCqVmcqTvjfe1Y/ec4WaRj/aQPvn+I=" crossorigin="anonymous" />
+		<link rel="stylesheet" href="../css/styles.css" />
+
+		<style></style>
+	</head>
+	<body>
+		<!-- Navigation bar and menu items are defined below -->
+		<nav>
+			<div class="top-bar" id="example-menu">
+				<div class="top-bar-left">
+					<ul class="dropdown menu" data-dropdown-menu>
+						<li class="menu-text">CityZen</li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+		<div class="row">
+			<div class="small-6 large-8 columns">
+				<!-- Populate the job posting details-->
+				<!-- Summary Section line one: Title: <h2>"Senior Full Stack Engineer</h2> -->
+				<!--Summary Section line two: Company Name | city, state | Remote or Onsite  -->
+
+				<!-- <hr> -->
+
+				<!-- Job Details: Heading with the title <h3>"Job Details"<h3> -->
+				<!-- Job Details: Heading with the Job Type <h4>"Job type" -->
+				<!-- Job Details: Heading with the Job Type <h4>"Job type" -->
+				<!-- Job Details: Paragraph stating full-time vs. part-time -->
+
+				<!-- <hr> -->
+
+				<!-- Job Benefits: Heading with the title <h3>"Benefits"<h3> -->
+				<!-- Job Benefits: <ul> of all the benefits provided by the company-->
+
+				<!-- <hr> -->
+				<!-- Full Job Description: Heading with the title <h3>"Full Job Description"<h3> -->
+				<!-- Full Job Description: <p>lorem em sem text </p>-->
+				<!-- Full Job Description:<br>-->
+        <!-- Full Job Description: <p>lorem em sem text </p> -->
+
+		<!--jQuery script tag-->
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<!-- Compressed JavaScript -->
+		<script src="https://cdn.jsdelivr.net/npm/foundation-sites@6.6.3/dist/js/foundation.min.js" integrity="sha256-pRF3zifJRA9jXGv++b06qwtSqX1byFQOLjqa2PTEb2o=" crossorigin="anonymous"></script>
+		<script src="../scripts/script.js"></script>
```

Comment:
Figured we'd add some initial pseudocode for the back button, save job button, and search appts. button.